src/components/MatPass.tsx

```
@@ -0,0 +1,478 @@
+    .filter(pkg => 
+      !pkg.name.toLowerCase().includes('invitado') &&
+      !pkg.name.toLowerCase().includes('hotel') &&
+      !pkg.name.toLowerCase().includes('co-work') &&
+      pkg.name.toLowerCase().includes('acro')
+    )
+    .sort((a, b) => a.price - b.price)
+    .map(pkg => {
+      const packageNumber = getPackageNumber(pkg.name);
+      const isDouble = pkg.price === 160 || pkg.price === 530;
+      // Asegurarse de que el ID sea un número válido
+      const validId = parseInt(pkg.id);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)